Network/dw-pihole.10s.py

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# <bitbar.title>Pi-hole Status</bitbar.title>
+# <bitbar.version>v1.0</bitbar.version>
+# <bitbar.author>Albert Steiner</bitbar.author>
+# <bitbar.author.github>alst-ovh</bitbar.author.github>
+# <bitbar.desc>Show Summary and Manage Pi-hole from Menubar.</bitbar.desc>
+# <bitbar.abouturl>https://github.com/alst-ovh/bitbar-dw-pihole/blob/master/dw-pihole.png</bitbar.abouturl>
```

Comment:
Could you add the image? I think `https://user-images.githubusercontent.com/35896287/86492400-5480c980-bd6e-11ea-83c1-081a11c3d63c.png` will do.